crew

```diff
@@ -717,6 +719,14 @@ def install
       # CREW_DEST_DIR contains usr/local/... hierarchy
       build_and_preconfigure target_dir
 
+      if @opt_check && Dir.exist?(target_dir)
+        Dir.chdir target_dir do
+          puts "Running post-build checks for package #{@pkg.name}..."
+          abort("#{@pkg.name} failed post-build checks. Aborting.".lightred) unless @pkg.check
+          puts "Checks for #{@pkg.name} have passed successfully.".lightgreen
+        end
+      end
+
```

Comment:
Maybe add this before the `resolve_dependencies_and_install` line in the `def install_package` function?
```
abort "The `-c` or `--check` option must also include either `-s` or `-S`".lightred if @opt_check and !@pkg.build_from_source
```

Reply:
I was wondering whether it would be worth it to enable checks for every source build if the `-S` option is present?